### ISSUE TITLE
Remove Nodes

### ISSUE THREAD

**Issue by NoYavy** [marmust/internet-scanner#17]
At some point it gets really cluttered, so maybe add a keybind to remove a single node and the option to reset instead of just relaunching

**Comment by marmust** [comment]
yes ill make it so you can remove a node, along with all of its "children"
thanks for the suggestion :)
